build.gradle

```diff
@@ -124,7 +124,7 @@ ext {
     androidxWorkVersion = "2.0.1"
 
     daggerVersion = '2.22.1'
-    fluxCVersion = '1.6.13'
+    fluxCVersion = '1.6.14'
```

Comment:
FYI there was an error in the WellSQL for 1.6.14 that was just discovered. Corrected that here:
https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1606

1.6.14 is only needed for https://github.com/wordpress-mobile/WordPress-Android/pull/12041 though, so we can either leave this as 1.6.13 or get the FluxC fix in either is fine IMO.